spec/abbreviato/abbreviato_spec.rb

```diff
@@ -201,7 +201,7 @@
     let(:cdata_example) { File.read('spec/fixtures/cdata_example.html') }
     it "cdata blocks are preserved" do
       text, truncated = Abbreviato.truncate(cdata_example, max_length: 65535)
-      expect(text.length).to eq 3583
+      expect(text.length).to eq 3581
```

Comment:
Leading and trailing spaces got striped

Reply:
@jwsf does seem ok? It jives with what @mesge found I think (the non-stripping with new nokogiri)

Reply:
I suspect this is unrelated to the [thing I found](https://github.com/rubys/nokogumbo/issues/64).